client/extensions/woocommerce/woocommerce-services/state/shipping-label/reducer.js

```diff
@@ -661,6 +662,32 @@ reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_PACKAGES ] = state => {
 	};
 };
 
+reducers.WCS_TOGGLE_DUMMY_FIELD = ( state, { value } ) => {
+	return {
+		...state,
+		form: {
+			...state.form,
+			customs: {
+				...state.form.customs,
+				dummyFieldChecked: value,
+			},
+		},
+	};
+};
+
+reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_SAVE_CUSTOMS ] = state => {
+	return {
+		...state,
+		form: {
+			...state.form,
+			customs: {
+				...state.form.customs,
+				submitted: true,
```

Comment:
What does `submitted` mean here?

Reply:
I think I'm all set here from your previous responses 👍 . This checks whether the customs step was submitted before, thus allowing only 'valid' or 'invalid' state after the first submission.